java-client/session/src/test/java/io/deephaven/client/impl/UpdateBySpecBuilderTest.java

```diff
@@ -35,8 +34,10 @@ public UpdateByColumn.UpdateBySpec visit(EmaSpec ema) {
             return UpdateByColumn.UpdateBySpec
                     .newBuilder().setEma(
                             UpdateByEma.newBuilder()
-                                    .setTimescale(UpdateByEmaTimescale.newBuilder().setTime(UpdateByEmaTime.newBuilder()
-                                            .setColumn("Timestamp").setPeriodNanos(1).build()).build())
+                                    .setTimescale(UpdateByEmaTimescale.newBuilder()
+                                            .setTime(UpdateByEmaTimescale.UpdateByEmaTime.newBuilder()
+                                                    .setColumn("Timestamp").setPeriodNanos(1).build())
+                                            .build())
```

Comment:
I would create some helper methods, `static UpdateByEmaTimescale time(String column, long nanos)` / `static UpdateByEmaTimescale tick(long ticks)`. Lot's of extraneous repetition IMO.